src/nodes/chunk_dispatch/chunk_dispatch.h

```diff
@@ -72,6 +72,17 @@ typedef struct ChunkDispatchState
 	 */
 	ChunkDispatch *dispatch;
 	ResultRelInfo *rri;
+	/*
+	 * Save attributes of targetList to calculate correct attribute no: in
+	 * case tuple has a different order incase of MERGE with INSERT
+	 */
+	List *colnames_list;
+	/*
+	 * Save dropped attributes to be used during execution to map
+	 * attributes numbers between different tuples.
+	 */
+	List *dropped_attrList;
+	List *chunk_resultrelInfoList;
```

Comment:
Can't find other usages of this one, is it needed?

Reply:
dropped_attrList is needed. I removed chunk_resultrelInfoList.